documentation/asciidoc/computers/remote-access/ssh.adoc

```diff
@@ -0,0 +1,176 @@
+[[ssh]]
+== Access a remote terminal with SSH
+
+You can access the terminal of a Raspberry Pi remotely from another computer on the same network using the **S**ecure **SH**ell (SSH) protocol.
+
+=== Enable the SSH server
+
+By default, Raspberry Pi OS disables the SSH server. Enable SSH in one of the following ways:
+
+==== On the desktop
+
+. From the *Preferences* menu, launch *Raspberry Pi Configuration*.
+. Navigate to the *Interfaces* tab.
+. Select *Enabled* next to *SSH*.
+. Click *OK*.
+
+==== From the terminal
+
+. Enter `sudo raspi-config` in a terminal window.
+. Select `Interfacing Options`.
+. Navigate to and select `SSH`.
+. Choose `Yes`.
+. Select `Ok`.
+. Choose `Finish`.
+
+==== Manually
+
+. Create an empty file named `ssh` in the boot partition:
++
+[source,console]
+----
+$ sudo touch /boot/firmware/ssh
+----
+. Reboot the machine:
++
+[source,console]
+----
+$ sudo reboot
+----
+
+=== Connect to an SSH server
+
+Open a terminal window on your computer and enter the following command, replacing the `<ip address>` placeholder with the xref:remote-access.adoc#ip-address[IP address of the Raspberry Pi you're trying to connect to] and `<username>` with your username:
+
+[source,console]
+----
+$ ssh <username>@<ip address>
+----
+
+When the connection works, you will see a security warning. Type `yes` to continue. You will only see this warning the first time you connect.
+
+Enter your account password when prompted.
+
+You should now see the Raspberry Pi command prompt:
+
+[source,console]
+----
+<username>@<hostname> ~ $
+----
+
+You are now connected to the Raspberry Pi remotely, and can execute commands.
+
+NOTE: If you receive a `connection timed out` error, you may have entered the wrong IP address for the Raspberry Pi. Check the xref:remote-access.adoc#ip-address[IP address of the Raspberry Pi].
+
+==== Forward X11 over SSH
+
+NOTE: On Raspberry Pi 4 and 5, Raspberry Pi OS Bookworm uses the Wayland window server by default. You can only forward X11 if you use the X window server. To enable window forwarding over X11, switch your desktop to the X window server in Raspberry Pi Configuration.
+
+NOTE: X11 is no longer installed by default on many desktop environments. Install a third-party X server such as https://www.xquartz.org/[XQuartz] to use X11 forwarding.
+
+X11 enables graphical applications over SSH. Pass the `-Y` flag to forward an X session over SSH:
+
+[source,console]
+----
+$ ssh -Y <username>@<ip address>
+----
+
+Once authenticated, you will see the command line as usual. However, you can also open graphical windows that an X server can render for you. For example, type the following command to launch a https://www.geany.org/[Geany] window:
+
+[source,console]
+----
+$ geany &
+----
+
+=== Configure SSH without a password
+
+To remotely access your Raspberry Pi without providing a password each time you connect, use an SSH keypair.
+
+==== Preconfigure a boot image with Raspberry Pi Imager
```

Comment:
I'm not sure that "boot image" is the right terminology here. "SD card" perhaps? :man_shrugging: